Sources/HAP/Utils/QRCode.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  QRCode.swift
+//  HAP
+//
+//  Created by Guy Brooker on 31/01/2018.
+//
```

Comment:
Please remove this file header comment

Reply:
Sorry, that is the default header from XCode. I'll remove. Would be nice to get a credit in the README for my contributions for Dynamic accessory management and auto-setup.

Reply:
Sure! Feel free to add your name to the README!